nixd/include/nixd/Server/Controller.h

```diff
@@ -122,6 +128,7 @@ class Controller : public lspserver::LSPServer {
 
   std::mutex ConfigLock;
 
+  configuration::TopLevel JSONConfig;
```

Comment:
maybe you can add some comment here, saying that it is the default value of config

Reply:
On second thoughts, I think I should call it `DefaultConfig` and then add a comment that says it is set to the contents of `.nixd.json` if it exists.